CHANGELOG.md

```diff
@@ -0,0 +1,48 @@
+# Changelog
+
+## 2.0.0
+
+ - SAML defaults to the SHA-256 hash signature hash
+ - An issue was fixed where absolute paths couldn't be used for certificates and
+   keys in YAML configuration.
+ - Better LDAP group syncing that isn't as likely to crash and leave the database
+   in an unknown state.
+ - Removes LDAP group mappings when a security group is deleted
+ - Minor UI changes for the login form
+ - Moved the AD fields in the CMS to a separate tab and made them read-only.
+ - Member "last synced" time is now showing the correct date and time.
+ - Adding a script to rotate LDAP binding credentials for system administrators
+ - Samba has been confirmed to work
+ - Automatic syncing of LDAP groups and users via the `LDAPAllSyncJob` build task
+ - Users can now reset their AD password via the "I've lost my password" on the
+   login form.
+ - Updated documentation
+
+LDAP functionality that modifies user data (i.e. password reset) will require
```

Comment:
Not sure at all, it's kind of messy since I tried to find relevant pieces in the git log.